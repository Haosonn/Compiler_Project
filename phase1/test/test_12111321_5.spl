int test_variable(int a, int b, int c) {
    float d = 1.0;
    char e = 'a';
    char f = '\xfff';
    char g = '12333';
}