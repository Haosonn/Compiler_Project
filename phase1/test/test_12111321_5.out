Error type A at Line 4: unknown lexeme '\xfff'
Error type A at Line 5: unknown lexeme '12333'
